### PR TITLE
chore(*): Add public signing key

### PR DESCRIPTION
**Pull request by hickeyma**
Add public key for signed releases

Part of: #6015

**What this PR does / why we need it**:
Public key needed for signed releases.

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

File: KEYS [hunk 1]
```diff
@@ -149,4 +149,62 @@ qdCDQHoUOgtZ4v6+QaxI3WV1KPBsPb7SAjuphubIQVK/6qHse9OoWVwWAABXHFqX
 +YMbPW0/IQ==
 =g11H
 -----END PGP PUBLIC KEY BLOCK-----
+pub   rsa4096 2019-05-15 [SC]
```
Comment (mattfarina) [inline KEYS]: Small nit... Can you add a blank line between `-----END PGP PUBLIC KEY BLOCK-----` and the start of your content

Comment (hickeyma) [reply KEYS]: Can do and thought about this. One thing though - would this blank line be added if you used the instruction from KEYS file per verbatim?

Comment (hickeyma) [reply KEYS]: Added the blank line

**Review by mattfarina — APPROVED** [review]

File: KEYS (hunk 1 above)
Comment (mattfarina) [reply KEYS]: > would this blank line be added if you used the instruction from KEYS file per verbatim?

@hickeyma I'm not entirely sure. Would need to dig more into this.

Comment (hickeyma) [reply KEYS]: That's ok. just asking.

**Pull request merged**